includes/fpm-cron-runner.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+//
+// This file is being templated and managed by Kubernetes wpvip-operator.
+//
+
+//
+// An init container will be needed that runs something along the lines of
```

Comment:
We can put the wpcli phar inside the chroot in the image build even. Or extract it, but see my other comment regarding that.